components/blocks/FilesToCards.vue

```diff
@@ -221,9 +246,12 @@ export default {
   }),
   computed: {
     cardTags() {
-      const tags = this.data.map((card) => card.group);
+      let tags = this.data.map((card) => card.tags);
+      const groups = this.data.map((card) => card.groups);
+      const languages = this.data.map((card) => card.languages);
+      tags = tags.concat(groups, languages).flat()
       return tags
-        .filter((group, index) => tags.indexOf(group) === index)
+        .filter((tag, index) => tags.indexOf(tag) === index)
         .sort();
```

Comment:
I would consider making this an array of objects with tag and type, so you can also use it in the tag display on the cards instead of three v-ifs

Reply:
Wait, this is getting all unique tags - mostly nevermind.  Though I would consider how to make the code less repetitive, maybe doing a v-for over the tag fields.

Reply:
i could change to a v-for with cardTags.... but then all tags would be on the same line essentially. currently with the v-ifs i can separate them into a line of tags for langauges, a line of tags for groups, etc. not sure how you feel about visually separating the different tag types versus lumping them all into the same pot.

Reply:
<img width="1725" alt="Screen Shot 2021-12-07 at 3 30 40 PM" src="https://user-images.githubusercontent.com/15157854/145101706-aeaf90ee-04d6-4508-ae32-094342f3d97f.png">
<img width="1680" alt="Screen Shot 2021-12-07 at 3 30 15 PM" src="https://user-images.githubusercontent.com/15157854/145101731-1bc144e6-20a6-4fb0-9307-aae77b78b161.png">

Reply:
I like the later styling, but I'd distinguish the group by color. So pick a palette of three colors to use - one for language, one for groups, one for tags.